action.yml

```diff
@@ -102,13 +103,13 @@ runs:
         retention-days: 5
 
     - name: Validate hyperlinks
+      if: ${{ inputs.VALIDATE_LINKS == "true" }}
```

Comment:
We want to do this for all such steps in future. Had https://github.com/w3c/spec-prod/issues/5 waiting for the `if:` support.

Reply:
From the schema, it does look like it's supported now https://json.schemastore.org/github-action.json

Reply:
Yes, it's so recent that it's not even documented: https://github.com/github/docs/pull/12378 😄